pkg/cluster/cluster_test.go

```diff
@@ -243,34 +169,15 @@ var _ = Describe("cluster.Cluster", func() {
 	})
 })
 
-var _ inject.Cache = &injectable{}
-var _ inject.Client = &injectable{}
 var _ inject.Scheme = &injectable{}
-var _ inject.Config = &injectable{}
 var _ inject.Logger = &injectable{}
 
 type injectable struct {
```

Comment:
Do we still need the injectable? Seems unused (except l.172+173)

Reply:
Thanks! Leftovers :)